cdap-app-fabric/src/main/java/co/cask/cdap/metadata/MetadataHttpHandler.java

```diff
@@ -76,7 +77,8 @@ public MetadataHttpHandler(MetadataAdmin metadataAdmin) {
   public void getAppMetadata(HttpRequest request, HttpResponder responder,
```

Comment:
So all the handlers make request with scope user. How can an user retrieve system metadata then if they want to see it ? o